eth/executionclient/execution_client_test.go

```diff
@@ -210,7 +133,7 @@ func TestFetchHistoricalLogs(t *testing.T) {
 }
 
 func TestStreamLogs(t *testing.T) {
-	const testTimeout = 1000 * time.Millisecond
+	const testTimeout = 10000 * time.Millisecond
```

Comment:
I think we need to make this test a bit shorter: Something that is enough to say that the test works